### PR TITLE
[DOC] Missing link to R DataFrame API doc

### PR DESCRIPTION
**Pull request by Lewuathe**

### CODE REVIEW

**Comment by felixcheung** [comment]
@shivaram

**Comment by shivaram** [comment]
On this note can we update the roxygen doc for DataFrames ? http://spark.apache.org/docs/latest/api/R/DataFrame.html seems pretty incomplete.

**Comment by felixcheung** [comment]
why don't we point to http://spark.apache.org/docs/latest/sparkr.html for now instead?

**Comment by Lewuathe** [comment]
@felixb Other docs for Scala, Java and Python toward API documents. So I thought it might be better to make it indicate toward http://spark.apache.org/docs/latest/api/R/DataFrame.html too.

**Comment by shivaram** [comment]
Yeah but the DataFrame.html doesn't seem very useful the way it is right now -- can you just update the roxygen docs in this PR as well ?

**Comment by Lewuathe** [comment]
@shivaram Sure. I'll submit.

**Comment by Lewuathe** [comment]
This failure does not seem to be related to the patch.

**Comment by shivaram** [comment]
Jenkins, retest this please

**Comment by shivaram** [comment]
Jenkins, retest this please

**Comment by shivaram** [comment]
I think this error is related
```
First time using roxygen2 4.0. Upgrading automatically...
Error: Failure in roxygen block beginning DataFrame.R:23
Mismatched braces: "@seealso \{createDataFrame}, \link{jsonFile}, \link{table}"
Execution halted
```

**Comment by felixcheung** [comment]
it should be `@seealso \link{createDataFrame}`

**Comment by shivaram** [comment]
The latest error is 
```
Error: Failure in roxygen block beginning DataFrame.R:23
@export may only span a single line
```

@Lewuathe you can verify this locally using `./R/create-docs.sh` before submitting to Jenkins

**Comment by Lewuathe** [comment]
@shagunsodhani Thank you But that error was not occurred on local. 
Anyway I can update it.

File: R/pkg/DESCRIPTION [hunk 1]
```diff
@@ -35,3 +35,4 @@ Collate:
     'sparkR.R'
     'stats.R'
     'utils.R'
+RoxygenNote: 5.0.0
```
Comment (shivaram) [inline R/pkg/DESCRIPTION]: I'm not sure we need to modify this file in this PR - It encodes some dependencies among R files which is sometimes fragile.

**Comment by shivaram** [comment]
Thanks @Lewuathe -- LGTM. This is good to go if you can revert the change to the `DESCRIPTION` file.

**Comment by shivaram** [comment]
Thanks again. Merging this

**Comment by Lewuathe** [comment]
@felixcheung  @shivaram Thank you so much!

**Pull request closed without merging**